var Admin_usuarios_model = Backbone.Model.extend({
    
    defaults: {
        /*formRegister: {}*/
    }
    /*, initialize: function () {
        try {
           alert();
        } catch (exception) {
            console.log(exception);
        }
    }*/
    ,f_guardarNuevoUsuario: function(arg_dataCollection){
        try{
            return $.ajax({
                url: BASE_URL+'admin-usuarios/nuevo-usuario',
                type: "POST",
                data: {"data": arg_dataCollection}
            });
        } catch (exception) {
            console.log(exception);
        }
    },
    f_borrarUsuario: function(id_persona){
        try{
            return $.ajax({
                url: BASE_URL+'admin-usuarios/borrar-usuario',
                type: "POST",
                data: {"id_persona": id_persona}
            });
        } catch (exception) {
            console.log(exception);
        }
    },
    f_cambiosUsuario: function(la_data){
        try{
            return $.ajax({
                url: BASE_URL+'admin-usuarios/editar-usuario',
                type: "POST",
                data: {"data": la_data}
            });
        } catch (exception) {
            console.log(exception);
        }
    },
    f_docs:function(){
        try{
            return $.ajax({
                url: BASE_URL+'admin-usuarios/docs-candidatos',
                type: "POST"
            });
        } catch (exception) {
            console.log(exception);
        }
    },
    f_asiganVacanteModel:function(la_data){
        try{
            return $.ajax({
                url: BASE_URL+'admin-usuarios/vacante-candidato',
                type: "POST",
                data: {"data": la_data}
            });
        } catch (exception) {
            console.log(exception);
        }
    },
    f_asiganDocs:function(la_data){
        try{
            return $.ajax({
                url: BASE_URL+'admin-usuarios/docs-candidato',
                type: "POST",
                data: {"data": la_data}
            });
        } catch (exception) {
            console.log(exception);
        }
    }

});